src/components/views/messages/ReactionsRowButtonTooltip.tsx

```diff
@@ -43,12 +43,16 @@ export default class ReactionsRowButtonTooltip extends React.PureComponent<IProp
         let tooltipLabel: JSX.Element | undefined;
         if (room) {
             const senders: string[] = [];
+            let customReactionName = "";
```

Comment:
```suggestion
            let customReactionName: string | undefined;
```

I think I'd prefer this but it's more of a personal preference